src/components/DatePicker/RelativeDateRange.jsx

```diff
@@ -258,4 +284,5 @@ RelativeDateRange.propTypes = {
     showApply: PropTypes.bool,
     value: PropTypes.string,
     timezoneName: PropTypes.string,
+    isFutureDatesAllowed: PropTypes.bool, // New prop type
```

Comment:
same // New prpo => maynot needed.
you can add comment of what it does, or add in Ui lit story board https://ui.xola.io/?path=/docs/data-display-date-time-date-range-picker--relative-date-ranges